ci-operator/config/openshift/ironic-image/openshift-ironic-image-master.yaml

```diff
@@ -0,0 +1,34 @@
+base_images:
+  base:
+    name: "4.2"
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.10
+images:
+- dockerfile_path: Dockerfile.ocp
+  from: base-os
+  to: ironic-image
+promotion:
+  name: "4.2"
+  namespace: ocp
+raw_steps:
+- pipeline_image_cache_step:
+    commands: |
+      curl http://base-openstack-4-2.ocp.svc > /etc/yum.repos.d/base-openstack-4-2.repo
```

Comment:
@smarterclayton what's going on here?

Reply:
we're injecting a new repository since we need rpms from the openstack repo

Reply:
and note it's already used in a couple other images:

```
$ git grep base-openstack
Makefile:       oc create secret generic base-openstack-4-2-repos \
Makefile:       oc create secret generic base-openstack-4-3-repos \
ci-operator/config/openshift/ironic-inspector-image/openshift-ironic-inspector-image-master.yaml:      curl http://base-openstack-4-2.ocp.svc > /etc/yum.repos.d/base-openstack-4-2.repo
ci-operator/config/openshift/ironic-ipa-downloader/openshift-ironic-ipa-downloader-master.yaml:      curl http://base-openstack-4-2.ocp.svc > /etc/yum.repos.d/base-openstack-4-2.repo
ci-operator/infra/openshift/release-controller/rpms-ocp-4.2.yaml:    name: base-openstack-4-2
ci-operator/infra/openshift/release-controller/rpms-ocp-4.2.yaml:      app: base-openstack-4-2
ci-operator/infra/openshift/release-controller/rpms-ocp-4.2.yaml:    name: base-openstack-4-2
ci-operator/infra/openshift/release-controller/rpms-ocp-4.2.yaml:      app: base-openstack-4-2
ci-operator/infra/openshift/release-controller/rpms-ocp-4.2.yaml:        app: base-openstack-4-2
ci-operator/infra/openshift/release-controller/rpms-ocp-4.2.yaml:          app: base-openstack-4-2
ci-operator/infra/openshift/release-controller/rpms-ocp-4.2.yaml:              secretName: base-openstack-4-2-repos
ci-operator/infra/openshift/release-controller/rpms-ocp-4.3.yaml:    name: base-openstack-4-3
ci-operator/infra/openshift/release-controller/rpms-ocp-4.3.yaml:      app: base-openstack-4-3
ci-operator/infra/openshift/release-controller/rpms-ocp-4.3.yaml:    name: base-openstack-4-3
ci-operator/infra/openshift/release-controller/rpms-ocp-4.3.yaml:      app: base-openstack-4-3
ci-operator/infra/openshift/release-controller/rpms-ocp-4.3.yaml:        app: base-openstack-4-3
ci-operator/infra/openshift/release-controller/rpms-ocp-4.3.yaml:          app: base-openstack-4-3
ci-operator/infra/openshift/release-controller/rpms-ocp-4.3.yaml:              secretName: base-openstack-4-3-repos
```